ttar

```diff
@@ -28,7 +28,7 @@ export LC_ALL=C
 
 path=""
 CMD=""
-ARG_STRING="$*"
+printf -v ARG_STRING '%q ' "$@"
```

Comment:
run_test.sh fails due to this change: the ARG_STRING now has a trailing space. We need to fix that or update the test.

Reply:
Can follow up with `ARG_STRING=${ARG_STRING% }` as a following command to trim that.